test/lld/bigint.wat

```diff
@@ -0,0 +1,11 @@
+(module
+ (table $0 1 1 funcref)
+ (elem (i32.const 1) $foo)
+ (export "__data_end" (global $global$1))
+ (global $global$0 (mut i32) (i32.const 66208))
+ (global $global$1 i32 (i32.const 658))
+ (func $foo (param i64) (result i64)
```

Comment:
I was expecting to see imports and exports of i64 functions here... is putting it the table enough to test the legalization?

Reply:
Yes, the legalization path would create a legalized dynCall with i32 pairs etc.